tests/src/main/python/rest/tests/utilities/TestClass.py

```diff
@@ -0,0 +1,38 @@
+import sys
+import os
+from sys import path
+from os import getcwd
+path.append(getcwd() + "/../../src/main/python/rest/tests/extract")
+import swagger_client
+import unittest
+import json
+
+
+class TestClass(unittest.TestCase):
+    #Connection Details
+    configuration = swagger_client.Configuration()
+    #Check if we should read from a json file or environment variable.
+    if os.environ['FHIR_JSON']=='False':
+        #Load from Environment.
+        configuration.username = os.environ['FHIRUSER']
+        configuration.password = os.environ['FHIRPASS']
+        fhir_endpoint = 'https://fhir-internal.dev:9443/fhir-server/api/v4'
+        fhir_tenant = 'Default'
+    else:
+        #Load from JSON file and set.
+        #config/local-ibm-fhir.json <- whats the real one? See create_pod shell script.
```

Comment:
I'm not sure I understand this comment.
Could you elaborate?

Reply:
Jai has better understanding of how we will actually load the json file and where the real one is at. So some additional massaging of this code is expected. My work was to get things started and Jai will pick it up and run with it.